api/envoy/extensions/filters/network/dubbo_proxy/v3/route.proto

```diff
@@ -27,7 +27,7 @@ message RouteConfiguration {
   string name = 1;
 
   // The interface name of the service.
-  string interface = 2;
+  type.matcher.v3.StringMatcher interface = 2;
```

Comment:
changing a type is an incompatible change. You have to deprecate the old field and add a new filed `interface_match` as `StringMatcher`.